rust/relay/src/main.rs

```diff
@@ -160,6 +160,8 @@ async fn main() -> Result<()> {
             }
         }
     } else {
+        tracing::info!("No portal token supplied, starting standalone mode");
```

Comment:
```suggestion
        tracing::warn!("No portal token supplied, starting standalone mode");
```

Apply this if you feel strongly.